packages/kbn-language-documentation-popover/src/components/shared/documentation_content.tsx

```diff
@@ -29,7 +31,12 @@ function DocumentationContent({
 }: DocumentationContentProps) {
   return (
     <>
-      <EuiFlexGroup gutterSize="none" responsive={false} direction="column">
+      <EuiFlexGroup
```

Comment:
@drewdaemon In am fixing a bit the CSS here based on your comment on the first PR